terraform/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "gcp_project_id" {
+  type        = string
+  description = "The GCP project ID to apply this config to."
+  default     = "<gcp-project-id>"    # Change this to your Google Cloud project ID
+}
+
+variable "apis" {
+    description = "List of Google Cloud APIs to be enabled"
+    type        = list(string)
+    default = [
+        "container.googleapis.com",
+        "monitoring.googleapis.com",
+        "cloudtrace.googleapis.com",
+        "clouddebugger.googleapis.com",
+        "cloudprofiler.googleapis.com"]
+}
+
+variable "name" {
+    type        = string
+    description = "Name of Cluster"
```

Comment:
`Name to be given to the newly created cluster`

I think we should be clear about whether this variable is used to name a new resource or a reference to an existing resource. For example if we were to be using an existing `network` that we are using then we might say `Name of the network to attache the cluster to`